dcp.bionimbus.org/etlMapping.yaml

```diff
@@ -1,62 +1,60 @@
-case:
-  _props:
-    - submitter_id
-    - project_id
-    - consent_codes
-  _flatten_props:
-    - path: demographics
-      _props:
-        - gender
-        - race
-        - ethnicity
-        - age_at_index
-        - bmi
-    - path: medical_histories
-      _props:
-        - diabetes
-        - hypertension
-        - cerebrovascular_disease
-        - chronic_respiratory_disease
-        - coronary_artery_disease
-        - asthma
-        - cabg_presence
-        - copd
-        - emphysema
-        - heart_failure
-        - myocardial_infarction
-        - stroke
-    - path: diagnoses
-      _props:
-        - metabolic_syndrome_diagnosis
-    - path: blood_pressure_test
-      _props:
-        - diastolic_blood_pressure
-        - systolic_blood_pressure
-    - path: medication_records
-      _props:
-        - hypertension_meds
-    - path: samples
```

Comment:
could you check if we still have this node in the dictionary, if yes, we should keep it in the new mapping